id-repository/credential-service/src/test/java/io/mosip/credentialstore/test/util/EncryptionUtilTest.java

```diff
@@ -181,7 +181,7 @@ public void encryptionWithZkFailureTest() throws JsonParseException, JsonMapping
 		Mockito.when(objectMapper.readValue(cryptoResponse, CryptoZkResponseDto.class)).thenReturn(cryptoZkResponseDto);
 		encryptionUtil.encryptDataWithZK("12345678", zkDataAttributeList, "requestId");
 	}
-	@Test(expected = DataEncryptionFailureException.class)
+	@Test

```

Comment:
As per test case, IOException needs to be handled and DataEncryptionFailureException needs to be thrown. Update test case to throw DataEncryptionFailureException 

original code:
`		} catch (IOException e) {
			LOGGER.error(IdRepoSecurityManager.getUser(), LoggerFileConstant.REQUEST_ID.toString(), requestId,
					"ZK encryption error with error message" + ExceptionUtils.getStackTrace(e));
			throw new DataEncryptionFailureException(IO_EXCEPTION, e);`